.github/workflows/gaia-tests.yml

```diff
@@ -3,31 +3,29 @@ name: TESTING - gaia to gaia integration
 on: [push]
 
 jobs:
-
   build:
     name: build
     runs-on: ubuntu-latest
     steps:
+      # Install and setup go
+      - name: Set up Go 1.14
+        uses: actions/setup-go@v1
+        with:
+          go-version: 1.14
+        id: go
 
-    # Install and setup go
-    - name: Set up Go 1.14
-      uses: actions/setup-go@v1
-      with:
-        go-version: 1.14
-      id: go
+      # checkout relayer
+      - name: checkout relayer
+        uses: actions/checkout@v2
 
-    # checkout relayer
-    - name: checkout relayer
-      uses: actions/checkout@v2
+      # build cache
+      - uses: actions/cache@v1
+        with:
+          path: ~/go/pkg/mod
+          key: ${{ runner.os }}-go-${{ hashFiles('**/go.sum') }}
+          restore-keys: |
+            ${{ runner.os }}-go-
 
-    # build cache
-    - uses: actions/cache@v1
-      with:
-        path: ~/go/pkg/mod
-        key: ${{ runner.os }}-go-${{ hashFiles('**/go.sum') }}
-        restore-keys: |
-          ${{ runner.os }}-go-
-    
-    # run tests
-    - name: run gaia tests
-      run: TEST_DEBUG=true make test-gaia
+      # run tests
+      - name: run gaia tests
+        run: make test-gaia
```

Comment:
```suggestion
        run: TEST_DEBUG=true make test-gaia
```

Reply:
isnt this redundant as its set in the make file as well?

Reply:
Ah! Thats why! Thank you!